backup-and-restore-using-dumpling-lightning.md

```diff
@@ -1,10 +1,16 @@
 ---
 title: 使用 Dumpling/TiDB Lightning 进行备份与恢复
-aliases: ['/docs-cn/dev/backup-and-restore-using-dumpling-lightning/','/docs-cn/dev/export-or-backup-using-dumpling/','/zh/tidb/dev/export-or-backup-using-dumpling']
+aliases: ['/docs-cn/dev/backup-and-restore-using-dumpling-lightning/','/docs-cn/dev/export-or-backup-using-dumpling/','/zh/tidb/dev/export-or-backup-using-dumpling','/docs-cn/dev/backup-and-restore-using-mydumper-lightning/','/docs-cn/dev/how-to/maintain/backup-and-restore/mydumper-lightning/','/docs-cn/dev/how-to/maintain/backup-and-restore/','/backup-and-restore-using-mydumper-lightning/']
```

Comment:
'/backup-and-restore-using-mydumper-lightning/' -> '/zh/tidb/dev/backup-and-restore-using-mydumper-lightning/'

Reply:
Done in https://github.com/pingcap/docs-cn/pull/5208.